src/actions.js

```diff
@@ -236,7 +234,6 @@ export function fetchUserMutation(mm, clientMutationId) {
   return graphql(payload, "ADMIN_USER");
 }
 
-
 export function fetchRegionDistricts(parent) {
   let filters = [`type: "D"`];
   if (!!parent) {
```

Comment:
Can you remove this double negation?

Reply:
or use something different that looks better than such piece of code?

Reply:
Double negation removed.